frontend/src/app/patients/EditPatient.test.tsx

```diff
@@ -129,4 +130,141 @@ describe("EditPatient", () => {
+                  employedInHealthcare: true,
+                  facility: null,
+                },
+              },
+            },
+          },
+          {
+            request: {
+              query: UPDATE_PATIENT,
+              variables: {
+                patientId: mockPatientID,
+                firstName: "Gina",
+                middleName: null,
+                lastName: "Franecki",
+                birthDate: "1939-10-11",
+                street: "736 Jackson PI NW",
+                streetTwo: null,
+                city: null,
+                state: "DC",
+                zipCode: "20503",
+                telephone: "(301) 458-4775",
+                role: null,
+                email: null,
+                county: null,
+                race: null,
+                ethnicity: null,
+                gender: null,
+                tribalAffiliation: null,
+                residentCongregateSetting: true,
+                employedInHealthcare: true,
+                facilityId: null,
+              },
+            },
+            result: {
+              data: {
+                internalId: "153f661f-b6ea-4711-b9ab-487b95198cce",
+              },
+            },
+          },
+        ];
+
+        component = render(
+          <MemoryRouter>
+            <Provider store={store}>
+              <MockedProvider mocks={mocks} addTypename={false}>
+                <EditPatient
+                  facilityId={mockFacilityID}
+                  patientId={mockPatientID}
+                />
+              </MockedProvider>
+            </Provider>
+            <Route
+              path="*"
+              render={({ location }) => {
+                testLocation = location;
+                return <div>Redirect</div>;
+              }}
+            />
+          </MemoryRouter>
+        );
+        await act(async () => {
+          await screen.findAllByText("Franecki, Eugenia", { exact: false });
+        });
+      });
+      it("can't be saved", () => {
+        expect(screen.getAllByText("Save changes")[0]).toBeDisabled();
+      });
+      describe("After field update", () => {
+        beforeEach(() => {
+          fireEvent.change(
+            screen.getByLabelText("First Name", {
+              exact: false,
+            }),
+            {
+              target: { value: "Gina" },
+            }
+          );
+        });
+        it("save is enabled", () => {
+          expect(screen.getAllByText("Save changes")[0]).not.toBeDisabled();
+        });
+        describe("On save", () => {
+          beforeEach(async () => {
+            await act(async () => {
+              fireEvent.click(
+                screen.queryAllByText("Save", {
+                  exact: false,
+                })[0]
+              );
+              // for some reason the 1 second time out works but this does not
+              // await screen.getAllByText("Redirect")
+              await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
`getAllByText` is synchronous, `findAllByText` returns a promise. Tested this locally, it should work.

```suggestion
              await screen.findAllByText("Redirect");
```